src/Backends/SystemUpdate.vala

```diff
@@ -189,7 +191,8 @@ public class SettingsDaemon.Backends.SystemUpdate : Object {
     }
 
     private void progress_callback (Pk.Progress progress, Pk.ProgressType progress_type) {
-        update_state (current_state.state, PkUtils.status_to_title (progress.status));
+        update_state (current_state.state, PkUtils.status_to_title (progress.status),
```

Comment:
If you have to split arguments into lines, a new line for every argument is preferred, see https://docs.elementary.io/develop/writing-apps/code-style#splitting-arguments-into-lines